mk/golang.mk

```diff
@@ -1,6 +1,9 @@
 # golang utilities
 GO_MIN_VERSION = 1.9
 
+# match Go's default GOPATH behaviour
+GOPATH ?= $(HOME)/go
```

Comment:
I would replace it with `export GOPATH := $(shell $(GOCC) env GOPATH)`

I am not 100% sure it works but it should.